django/db/models/fields/__init__.py

```diff
@@ -2213,6 +2213,7 @@ def __init__(
         name=None,
         protocol="both",
         unpack_ipv4=False,
+        error_messages=None,
```

Comment:
There is no need to modify a signature a pull `error_messages` outside of `kwargs`.